cmd/loop/staticaddr.go

```diff
@@ -624,6 +640,47 @@ func staticAddressLoopIn(ctx *cli.Context) error {
 	return nil
 }
 
+// selectDeposits sorts the deposits by amount in descending order, then by
+// blocks-until-expiry in ascending order. It then selects the deposits that
+// are needed to cover the amount requested without leaving a dust change. It
+// returns an error if the sum of deposits minus dust is less than the requested
+// amount.
+func selectDeposits(deposits []*looprpc.Deposit, targetAmount int64) ([]string,
```

Comment:
(see my next comment too)

Ideally this should be solved with a knapsack algorithm, but since that's a bit difficult, maybe we can use a variance of this heuristic to first try to find a covering deposit?